data/archipelago/scripts/patches/ap_extend_pixel_scenes.lua

```diff
@@ -0,0 +1,19 @@
+local nxml = dofile_once("data/archipelago/lib/nxml.lua")
+
+local scenePaths = {
+	"data/biome/_pixel_scenes.xml",
+	"data/biome/_pixel_scenes_newgame_plus.xml",
+}
+
+function APExtendPixelScenes()
+	for _, scenePath in ipairs(scenePaths) do
+
+		local content = ModTextFileGetContent(scenePath)
+		local xml = nxml.parse(content)
+		xml:first_of("PixelSceneFiles"):add_child(nxml.parse([[<File>data/archipelago/biome_impl/ap_egg.xml</File>]]))
+		ModTextFileSetContent(scenePath, tostring(xml))

```

Comment:
This should work with [this version of nxml](https://github.com/NathanSnail/luanxml).

```suggestion
		for xml in nxml.edit_file(scenePath) do
				xml:first_of("PixelSceneFiles"):add_child(nxml.parse([[<File>data/archipelago/biome_impl/ap_egg.xml</File>]]))
		end
```

Might want a check for whether PixelSceneFiles exists, other mods can do some wonky things and exclude any tag from its replacement scene.